### DISCUSSION TITLE
meshroom

### DISCUSSION THREAD

**Discussion by maxidesign34** [community/community#60460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

the current meshroom dose not work on my computer the older versions use to work but nothing works now after i opend the new versiom 2023.2.0. I have made some realy nice 3d models in older vertions but now they dont work so disappointing to see that this has come to THE END as a free sotfware i say this because peaple seem to think it operators, LET ME TELL YOU its not me and its not my pc  and i can say that it is the changes to software sorry the software is a big flop i am not going to fiddle with setting all-day  and week when it states on Alice Vision web sit the software will work of the bat with out doing any thing and still im not getting any results waste of my time.
![OBJ AFTER MESHING](https://github.com/community/community/assets/59597136/f4c0f475-8070-49e4-870e-4739f41beafa)
![MUSIC BUNNY](https://github.com/community/community/assets/59597136/c93f3637-8e09-4597-9807-130160927862)
maxidesign34@hotmail.com.

**Comment by AOHhiroko** [comment]
I do not understand what is going on, So give me time.

**Comment by maxidesign34** [comment]
AOHhiroko, thankyou for your reply its so frustrating to have all of the old programs not work neither  i can give you the reconstructed 3d mesh from the old programs when they where working so you can have a look. these are the first one i did with the old 2021.1.10 but now even those will not come out again like these ones.
![Buddha scan](https://github.com/community/community/assets/59597136/5606e404-b0b9-4a9f-8cf4-4f88af827f5e)
![gnome](https://github.com/community/community/assets/59597136/cf08602e-9715-4c3b-a520-0e912ef55656)
![green frog](https://github.com/community/community/assets/59597136/fac2afa2-e239-4d98-bb24-7ac3e37e4d09)
regards
maxigesign34@hotmail.com
